kadena/docs/src/design/security.md

```diff
@@ -0,0 +1,17 @@
+# Security considerations
+
+## Sphinx
+
+The [Sphinx](https://github.com/argumentcomputer/sphinx) prover is a fork of [SP1](https://github.com/succinctlabs/sp1)
+and as such inherits a lot from its security design. The current release of Sphinx (`dev`) has backported all the
+upstream security fixes as of SP1 `v1.0.8-testnet`. We will continue to update Sphinx with backports of upstream
+security fixes and subsequent updates to both Sphinx and the Light Client making them available as hotfixes.
+
+In terms of Sphinx-specific changes that require special attention, here is a non-exhaustive list of Sphinx
+AIR chips used for pre-compiles that are either not present in upstream SP1, or have had non-trivial changes:
+- `FieldAddChip`, `FieldSubChip`, `FieldMulChip`: Chips for BLS12-381 Fp acceleration.
+- `QuadFieldAddChip`, `QuadFieldSubChip`, `QuadFieldMulChip`: Chips for BLS12-381 Fp2 acceleration.
+- `Bls12381G1DecompressChip`: Chip for decompressing BLS12-381 compressed G1 points.
+- `Secp256k1DecompressChip`: Chip for decompressing K256 compressed points.
```

Comment:
We should mention the Blake2s and Sha512 chips too in this list, though I suppose we don't have to add them here until we update Sphinx with them

Reply:
Yep, will be part of the team TODO list that I'll share